Gemfile

```diff
@@ -3,11 +3,8 @@ source "https://rubygems.org"
 gemspec
 
 group :test do
-  gem "mocha"
-  gem "minitest"
-  gem "minitest-around"
-  gem "minitest-reporters"
   gem "rack-test"
```

Comment:
I have no idea why this would be here. I'm running the suite locally now without it and will commit its removal if it's green, or (hopefully) explain why it's here.